src/openrct2-ui/windows/RideList.cpp

```diff
@@ -402,6 +394,11 @@ namespace OpenRCT2::Ui::Windows
                 Invalidate();
         }
 
+        void OnPeriodicUpdate() override
```

Comment:
Can we retain the comment explaining why we only do it every so often?

Reply:
It's `RideHasAnyTrackElements` now

Reply:
No longer relevant, as we only sort now, rather than doing a full refresh. (A full refresh is still done when prompted to do so via an Intent, e.g. when adding a ride.)

Reply:
I see, thanks